src/resource/script.rs

```diff
@@ -193,6 +193,7 @@ declare_type_enum! {
     Text => "text",
     Items => "items",
     Number => "number",
+    // DateTime => "date-time",
```

Comment:
Unless you think somebody actually needs to return date/time values in the output fields of BigML scripts, let's just delete this line completely, since it isn't based on any docs or real-world observations.